docs/windows.md

```diff
@@ -222,12 +222,25 @@ kubectl apply -f -
 
 ##### 1. (Optional) Install OVS (provided by Antrea or your own)
 
+Antrea supports running OVS on Windows as native services or inside a
+host-process container. If you have an OVS package with a signed kernel
+driver and want to run OVS inside container, you can skip this step.
```

Comment:
The OVS package ships as part of the container image, right? So how do users know which one they have?

Reply:
The truth is antrea has delivered an image with unsigned driver as we mentioned later. But if user is using their own image, it may be packaged with a signed driver?

Reply:
Yes , if user wants to use their own image and it has a signed driver , they do not need to run the install-ovs script on the worker node